samples/dashboards/demos/climate/demo.js

```diff
@@ -179,32 +185,11 @@ async function setupBoard() {
                     enabled: false
                 },
                 series: [{
-                    // type: 'spline',
                     name: 'Timeline',
-                    data: (function () {
-                        const dateEnd = new Date(Date.UTC(2010, 11, 25)),
-                            dates = [];
-
-                        let date = new Date(Date.UTC(1951, 0, 5));
-
-                        while (date <= dateEnd) {
-                            dates.push([date.getTime(), 0]);
-                            date = date.getUTCDate() >= 25 ?
-                                new Date(Date.UTC(
-                                    date.getFullYear(),
-                                    date.getUTCMonth() + 1,
-                                    5
-                                )) :
-                                new Date(Date.UTC(
-                                    date.getFullYear(),
-                                    date.getUTCMonth(),
-                                    date.getUTCDate() + 10
-                                ));
-                        }
-
-                        return dates;
-                    }()),
-                    showInNavigator: false,
+                    data: [
+                        [Date.UTC(1951, 0, 5), 0],
```

Comment:
how about using something similar to `activeSelection` but like `dataExtremes`?

Reply:
It is now `activeTimeRange` because of the component-id `time-range-selector`.